### PR TITLE
ADEN-6590 Add UAP:HiVi fandom desktop tests

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomDesktop.java [hunk 1]
```diff
@@ -1,171 +1,111 @@
...
-  @Test
-  public void shouldHaveCorrectAspectRatioForImpactState() {
-    AdsFandomObject fandomPage = loadArticle(FandomAdsDataProvider.PAGE_HIVI_UAP_ARTICLE);
-    fandomPage.waitForPageLoad();
+  private TestAdsUapHiVi test() {
+    return test(FandomAdsDataProvider.PAGE_HIVI_UAP_ARTICLE);
+  }
 
-    assertAspectRatio(driver.findElement(TLB_SELECTOR).getSize(), DEFAULT_STATE_ASPECT_RATIO);
+  private TestAdsUapHiVi test(String page) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomDesktop.java]: MAJOR Remove this unused private "test" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [TestAdsUapHiViFandomMobile.java#L34](https://github.com/Wikia/selenium-tests/blob/4bed86fa058fab4e6b405d1721f83ce23d085139/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomMobile.java#L34): Define a constant instead of duplicating this literal "gpt-top-leaderboard" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [TestAdsUapHiViFandomMobile.java#L188](https://github.com/Wikia/selenium-tests/blob/4bed86fa058fab4e6b405d1721f83ce23d085139/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomMobile.java#L188): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)
1. ![MINOR][MINOR] [TestAdsUapHiViMobileWiki.java#L40](https://github.com/Wikia/selenium-tests/blob/4bed86fa058fab4e6b405d1721f83ce23d085139/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViMobileWiki.java#L40): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)
1. ![MINOR][MINOR] [TestAdsUapHiViMobileWiki.java#L186](https://github.com/Wikia/selenium-tests/blob/4bed86fa058fab4e6b405d1721f83ce23d085139/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViMobileWiki.java#L186): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomDesktop.java (hunk 1 above)
Comment (nandy-andy) [reply src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomDesktop.java]: Sonar it's being used: look below...

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [TestAdsUapHiViFandomMobile.java#L34](https://github.com/Wikia/selenium-tests/blob/37dd1d8411f2c8d39d52954c3ceafed79449fa7b/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomMobile.java#L34): Define a constant instead of duplicating this literal "gpt-top-leaderboard" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [TestAdsUapHiViFandomMobile.java#L188](https://github.com/Wikia/selenium-tests/blob/37dd1d8411f2c8d39d52954c3ceafed79449fa7b/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomMobile.java#L188): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)
1. ![MINOR][MINOR] [TestAdsUapHiViMobileWiki.java#L40](https://github.com/Wikia/selenium-tests/blob/37dd1d8411f2c8d39d52954c3ceafed79449fa7b/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViMobileWiki.java#L40): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)
1. ![MINOR][MINOR] [TestAdsUapHiViMobileWiki.java#L186](https://github.com/Wikia/selenium-tests/blob/37dd1d8411f2c8d39d52954c3ceafed79449fa7b/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViMobileWiki.java#L186): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**